e2e/karma.conf.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function getTestFiles(argv) {
+  const files = [];
+  if (argv.includes('--compat')) {
+    files.push('./tests/compat.test.ts');
+  } else if (argv.includes('--exp')) {
```

Comment:
`exp` is outdated now, and should be replaced throughout the PR.

Reply:
Done.